packages/e2e/playwright.config.ts

```diff
@@ -14,7 +14,7 @@ export default defineConfig({
   expect: {
     timeout: 15_000,
   },
-  maxFailures: 0,
```

Comment:
Why this change?

I mean I actually prefer it to be like this `1` so that if one e2e test failed the workflow stops until get fixed. What do you think?

Reply:
It's actually 1 because 0 disables the behaviour you described above.